tox.ini

```diff
@@ -12,6 +12,7 @@ commands =
 deps =
   -rrequirements/testing.txt
   defusedxml: defusedxml==0.4.1
+  py26: importlib
```

Comment:
+1 on dropping python 2.6 support if it makes anything easier. We could do that in this PR or in a separate PR and then bump the major version for the next release.